plugins/versionpress/src/Database/WpdbMirrorBridge.php

```diff
@@ -49,6 +49,13 @@ function insert($table, $data) {
 
         $entityName = $entityInfo->entityName;
         $data = $this->vpidRepository->replaceForeignKeysWithReferences($entityName, $data);
+
+        array_walk($data, function (&$key, $value) {
+            if ($value === false) {
+                $key = '';
```

Comment:
This is weird.

1. The signature of callback of `array_walk` is ($value, $key).
2. Variable `$value` (which represents key in this case) cannot be `false`.